Include/cpython/pystats.h

```diff
@@ -120,6 +120,10 @@ typedef struct _optimization_stats {
     uint64_t trace_length_hist[_Py_UOP_HIST_SIZE];
     uint64_t trace_run_length_hist[_Py_UOP_HIST_SIZE];
     uint64_t optimized_trace_length_hist[_Py_UOP_HIST_SIZE];
+    uint64_t optimizer_attempts;
+    uint64_t optimizer_successes;
+    uint64_t optimizer_failure_reason_null_function;
```

Comment:
Is this used? If it is, can you either rename it, or add a comment explaining what it means.